include/evse_security/evse_security.hpp

```diff
@@ -142,6 +142,12 @@ class EvseSecurity {
     /// @param ocsp_response the actual OCSP data
     void update_ocsp_cache(const CertificateHashData& certificate_hash_data, const std::string& ocsp_response);
 
+    // TODO: Switch to path
```

Comment:
Looks like this already returns a optional path, is that what this comment meant?

Reply:
Opps, simply forgot a line.